packages/launcher/static/applications/onyx-stats/README.md

```diff
@@ -0,0 +1,2444 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Could you edit or remove this file please? We probably don't need to full CRA instructions here.